src/ios/addPBXGroup.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Gets main project group that represents what you usually
+ * see in your left-side panel.
+ *
+ * This is done by accessing `PBXProjectSection` that
+ * by the `xcode` library design returns an object with keys:
+ * `uuid` & `uuid_comment`. We only access the first key in the array
+ * (just like `xcode` internally) and return that group details from `PBXGroup`
+ * section.
+ */
+const getMainGroup = (project) => {
+  const projectSection = project.pbxProjectSection();
+  const projectId = Object.keys(projectSection)[0];
```

Comment:
I am not comfortable with that statement to be fair, so I'd consider better solution to that.